.vsts-ci.yml

```diff
@@ -0,0 +1,155 @@
+resources:
+- repo: self
+  clean: true
+queue:
+  name: VSEng-MicroBuildVS2017
+  timeoutInMinutes: 360
+  demands: 
+  - msbuild
+  - visualstudio
+  - DotNetFramework
+
+variables:
+  BuildPlatform: 'Any CPU'
+steps:
+- task: NuGetCommand@2
+  inputs:
+    command: custom
+    arguments: 'locals all -clear'
+
+- task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+  inputs:
+    signType: real
+  condition: and(succeeded(), in(variables['PB_SignType'], 'test', 'real'))
+
+- task: ms-vseng.MicroBuildTasks.32f78468-e895-4f47-962c-58a699361df8.MicroBuildSwixPlugin@1
+  inputs:
+    feedSource: 'https://devdiv-test.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+- task: NuGetRestore@1
+  inputs:
+    solution: 'build\ToolsetPackages\InternalToolset.csproj'
+    feed: '8f470c7e-ac49-4afe-a6ee-cf784e438b93'
+
+- task: CmdLine@1
+  inputs:
+    filename: mkdir
+    arguments: 'Binaries\$(BuildConfiguration)'
+
+- task: VSBuild@1
+  inputs:
+    solution: 'src/Tools/MicroBuild/Build.proj'
+    vsVersion: 15.0
+    msbuildArgs: >- 
+        /p:TreatWarningsAsErrors=true 
+        /p:DeployExtension=false 
+        /p:TrackFileAccess=false
+        /p:OfficialBuild=true 
+        /p:VisualStudioVersion=14.0 
+        /flp1:Summary;Verbosity=diagnostic;Encoding=UTF-8;LogFile=$(Build.SourcesDirectory)\Binaries\$(BuildConfiguration)\Roslyn.log 
+        /flp2:WarningsOnly;Verbosity=diagnostic;Encoding=UTF-8;LogFile=$(Build.SourcesDirectory)\Binaries\$(BuildConfiguration)\Roslyn.wrn 
+        /flp3:ErrorsOnly;Verbosity=diagnostic;Encoding=UTF-8;LogFile=$(Build.SourcesDirectory)\Binaries\$(BuildConfiguration)\Roslyn.err 
+        /p:RoslynMyGetApiKey=$(Roslyn.MyGetApiKey) 
+        /p:RoslynNuGetApiKey=$(Roslyn.NuGetApiKey) 
+        /p:RoslynGitHubEmail=$(Roslyn.GitHubEmail) 
+        /p:RoslynGitHubToken=$(Roslyn.GitHubToken) 
+        /p:RoslynGitHubUserName=$(Roslyn.GitHubUserName) 
+        /p:PB_PublishBlobFeedKey=$(PB_PublishBlobFeedKey) 
+        /p:PublishStableVersions=false 
+        /p:VersionStampToPublish=prerelease'
```

Comment:
Is the quote at the end of the line expected to be there?

Reply:
Nope. That got missed somehow. Also doesn't seem to impact our build strangely.